ui/app/src/utils/xml.ts

```diff
@@ -126,7 +126,7 @@ export function createElements(doc: XMLDocument, element: Element, data: object)
         console.error('[createElements] Path not implemented.');
       } else if (content instanceof Element) {
         elem.appendChild(content);
-      } else {
+      } else if (content) {
```

Comment:
This might be an issue with `0` or `""`, so best to be specific
```suggestion
      } else if (content !== null && content !== void 0) {
```